hackerproject.js

```diff
@@ -1,11 +1,17 @@
-
+//Function is used to check if a property exists in a object and check if the property has certain value.
 var person={
     name:"Krithika",
     age:23,
     adress:"Mangalore",
     profession:"Trainee",
 }
 
+/**
```

Comment:
I think it would be great if we can have some comments explaining what the parameters key and value are.

Reply:
Fixed.